.github/workflows/net-perf-gke.yaml

```diff
@@ -313,6 +313,25 @@ jobs:
           artifacts: ./output/*
           other_files: cilium-sysdump-final.zip
```

Comment:
ohh because we export the sysdump to the bucket we don't have to merge the artifacts. Got it, nice catch!